ethcore/src/client/client.rs

```diff
@@ -1641,7 +1641,7 @@ impl BlockChainClient for Client {
 		self.chain.read().logs(blocks, |entry| filter.matches(entry), filter.limit)
 	}
 
-	fn filter_traces(&self, filter: TraceFilter) -> Option<Vec<LocalizedTrace>> {
+	fn filter_traces(&self, filter: TraceFilter, after: Option<&TraceId>, count: Option<u64>) -> Option<Vec<LocalizedTrace>> {
```

Comment:
Actually there were 3 options how to add this here:
- Modify TraceFIlter
- Add optional parameters to filter_traces
- Instead add new method like filter_traces_with_pagination
I've discussed it with @arkpar  and we chosen the second one. But frankly all of them pretty similar. There is no decisive advantage for any of them.